packages/cli/generators/project/templates/tsconfig.json.ejs

```diff
@@ -31,8 +31,8 @@
   },
   "include": ["src"],
   "exclude": [
-    "node_modules/**",
-    "packages/*/node_modules/**",
+    "**/node_modules/**",
```

Comment:
I think here it should still be "/node_modules/**", since the loopback app has `node_modules` as the root level folder.

Reply:
Please check this [glob example](https://globster.xyz/?q=**%2Fnode_modules%2F**&f=%2Fnode_modules%2Fmodule1%2Ffile.ts%2C%2Fnode_modules%2Fmodule2%2Ffile.ts%2C%2Fmyapp%2Fnode_modules%2Fmodule3%2Ffile.ts%2C%2Fmyapp%2Fnode_modules%2Fmodule4%2Ffile.ts%2C%2Fmyapp%2Fsrc%2Findex.ts) and let me know if it is correct.

Reply:
I see.